postgres/src/main/java/com/google/cloud/sql/postgres/SocketFactory.java

```diff
@@ -32,6 +35,9 @@
  */
 public class SocketFactory extends javax.net.SocketFactory {
   private static final Logger logger = Logger.getLogger(SocketFactory.class.getName());
+  private static final String GaeRuntime = "GAE_RUNTIME";
```

Comment:
Since this constant is only used in one place and is named the same as its contents, is it needed? Wouldn't it be clearer to just use the string literal?